server/src/main/java/io/spine/server/entity/RecordBasedRepository.java

```diff
@@ -426,7 +428,7 @@ protected E findOrCreate(I id) {
 
     private E findOrThrow(I id) {
         return find(id).orElseThrow(() -> newIllegalArgumentException(
-                "An entity with ID `%s` is not found in the repository.", id
+                "An entity `%s` with ID `%s` is not found in the repository.", entityClass(), id
```

Comment:
Please make sure the word `class` is not added to the message before the class name.